src/server/services/simple-connect/klachten.test.ts

```diff
@@ -0,0 +1,114 @@
+      },
+      {
+        klacht_inbehandeling: { value: '18-05-2022 13:48' },
+        klacht_datumontvangstklacht: { value: '05-05-2022' },
+        klacht_omschrijving: {
+          value: 'Dear Amsterdam Municipality, \r\nI have received a fine.',
+        },
+        klacht_gewensteoplossing: { value: '' },
+        klacht_klachtonderwerp: { value: '' },
+        klacht_id: { value: '36046' },
+        klacht_locatieadres: { value: '' },
+      },
+      {
+        klacht_inbehandeling: { value: '18-05-2022 13:47' },
+        klacht_datumontvangstklacht: { value: '06-05-2022' },
+        klacht_omschrijving: {
+          value:
+            'Geachte mevrouw, meneer,\r\n\r\n maar zij verwezen ons naar de gemeente.',
+        },
+        klacht_gewensteoplossing: { value: '' },
+        klacht_klachtonderwerp: { value: '' },
+        klacht_id: { value: '36044' },
+        klacht_locatieadres: { value: '' },
+      },
+      {
+        klacht_inbehandeling: { value: '18-05-2022 13:45' },
+        klacht_datumontvangstklacht: { value: '06-05-2022' },
+        klacht_omschrijving: {
+          value:
+            'Geachte mevrouw, meneer, We hebben een klacht ingediend bij de gemeente.',
+        },
+        klacht_gewensteoplossing: { value: '' },
+        klacht_klachtonderwerp: { value: '' },
+        klacht_id: { value: '36042' },
+        klacht_locatieadres: { value: '' },
+      },
+    ],
+  };
+
+  afterAll(() => {
+    // Enable http requests.
+    nock.enableNetConnect();
+    nock.restore();
+  });
+
+  beforeAll(() => {
+    // Disable real http requests.
+    // All requests should be mocked.
+    nock.disableNetConnect();
+  });
+
+  beforeEach(() => {
+    nock('http://localhost').post('/smile').reply(200, apiResponse);
+  });
+
+  afterEach(() => {
+    nock.cleanAll();
+    jest.clearAllMocks();
+  });
+
+  describe('transformKlachtenResponse', () => {
+    it('should transform the data correctly', () => {
+      const res = transformKlachtenResponse(apiResponse);
+
+      expect(res.klachten.length).toEqual(apiResponse.List.length);
+
+      expect(res.klachten[2]).toMatchInlineSnapshot(`
+        Object {
+          "gewensteOplossing": null,
+          "id": "36044",
+          "inbehandelingSinds": "2022-05-18T00:00:00.000Z",
+          "link": Object {
+            "title": "Klacht 36044",
+            "to": "/klachten/klacht/36044",
+          },
+          "locatie": null,
+          "omschrijving": "Geachte mevrouw, meneer,
+
+         maar zij verwezen ons naar de gemeente.",
+          "onderwerp": null,
+          "ontvangstDatum": "2022-05-06T00:00:00.000Z",
+        }
+      `);
+    });
+
+    it('should return data in expected format', async () => {
+      const res = await fetchKlachten('456', {
+        profile: {
+          id: '123',
+          authMethod: 'digid',
+          profileType: 'private',
+        },
+        token: 'abc123',
+      });
+
+      expect(res).toMatchSnapshot();
```

Comment:
Mits de snapshot niet te groot is kun je ook deze gebruiken:
```suggestion
      expect(res).toMatchInlineSnapshot();
```
werkt net wat lekkerder.